spring-cloud-contract-tools/spring-cloud-contract-spec-pact/src/test/groovy/org/springframework/cloud/contract/verifier/spec/pact/PactContractConverterSpec.groovy

```diff
@@ -185,12 +193,18 @@ class PactContractConverterSpec extends Specification {
           "something": "foo"
         },
         "matchingRules": {
-          "$.body.id": {
-            "match": "regex",
-            "regex": "[0-9]{3}"
-          },
-          "$.body.something": {
-            "match": "equality"
+          "body": {
```

Comment:
maybe we should check for the headers matchers too?

Reply:
Yes, that's for some time later. They also introduced path & query matchers.
See https://github.com/pact-foundation/pact-specification/tree/version-3#drop-the-jsonpath-notation-for-matchers